redisearch/client.go

```diff
@@ -148,6 +148,16 @@ type IndexingOptions struct {
 	Replace  bool
 }
 
+// MultiError Represents one or more errors
+type MultiError map[int]error
+
+func (e MultiError) Error() string {
```

Comment:
I put newlines, we can always make it look like an array. It's just for debug printing etc.